grails-shell/src/main/groovy/org/grails/cli/profile/repository/MavenProfileRepository.groovy

```diff
@@ -71,23 +72,37 @@ class MavenProfileRepository extends AbstractJarProfileRepository {
         return super.getProfile(profileShortName)
     }
 
-    protected Profile resolveProfile(String profileName) {
-        def defaultProfileVersion = BuildSettings.isDevelopmentGrailsVersion() ? 'LATEST' : BuildSettings.grailsVersion
-        if (!profileName.contains(':')) {
-            profileName = "org.grails.profiles:$profileName:$defaultProfileVersion"
+    protected DefaultArtifact resolveProfileArtifact(String profileName) {
+        if (profileName.contains(':')) {
+            return new DefaultArtifact(profileName)
         }
-        def art = new DefaultArtifact(profileName)
-        def groupId = art.groupId ?: 'org.grails.profiles'
-        def artifactId = art.artifactId
-        def version = art.version ?: defaultProfileVersion
+
+        def artifactId = profileName
+        def groupId = "org.grails.profiles"
+        def version = BuildSettings.isDevelopmentGrailsVersion() ? 'LATEST' : BuildSettings.grailsVersion
+
+        Map<String, Map> defaultValues = GrailsCli.getSetting("grails.profiles", Map, [:])
+        defaultValues.remove("repositories")
```

Comment:
It's not necessary to do so, however the data for the profiles will sit on the same level as `repositories`. So I removed it to prevent any possible issues with someone typing `grails create-app abc -profile repositories`